vue-app/src/App.vue

```diff
@@ -325,6 +325,8 @@ summary:focus {
   padding: 1.5rem;
   width: $cart-width-open;
   height: 100%;
+  position: sticky;
+  top: 0;
```

Comment:
We could use `top: 1.5rem;` (the same padding amount => line#325) to avoid cutting the top image we have on the sidebar when scrolling.

Reply:
Thanks!